Marlin/src/module/ft_motion/stepping.cpp

```diff
@@ -101,7 +101,7 @@ uint32_t Stepping::plan() {
   // The reciprocal is actually 2^32/dividend, but that requires dividing a uint64_t, which quite expensive
   // Since even the real reciprocal may underestimate the quotient by 1 anyway already, this optimisation doesn't
   // make things worse. This underestimation is compensated for in advance_until_step.
-  #define _DIVIDEND_RECIP(A) advance_dividend_reciprocal.A = UINT32_MAX / stepper_plan.advance_dividend_q0_32.A;
+  #define _DIVIDEND_RECIP(A) (stepper_plan.advance_dividend_q0_32.A ==0) ? advance_dividend_reciprocal.A = UINT32_MAX : advance_dividend_reciprocal.A = UINT32_MAX / stepper_plan.advance_dividend_q0_32.A;
```

Comment:
```suggestion
  #define _DIVIDEND_RECIP(A) advance_dividend_reciprocal.A = (stepper_plan.advance_dividend_q0_32.A == 0) ? UINT32_MAX : UINT32_MAX / stepper_plan.advance_dividend_q0_32.A;
```

Reply:
Or 

```suggestion
  #define _DIVIDEND_RECIP(A) advance_dividend_reciprocal.A = UINT32_MAX / (stepper_plan.advance_dividend_q0_32.A == 0 ? 1 : stepper_plan.advance_dividend_q0_32.A);
```

Both the same as yours but shorter.

Although divide by one may be slower than not doing it, up to you.